packages/ERTP/src/displayInfo.js

```diff
@@ -40,3 +41,12 @@ export const assertDisplayInfo = allegedDisplayInfo => {
   const displayInfoKeys = harden(['decimalPlaces']);
   assertKeysAllowed(displayInfoKeys, allegedDisplayInfo);
 };
+
+export const coerceDisplayInfo = allegedDisplayInfo => {
+  if (passStyleOf(allegedDisplayInfo) === REMOTE_STYLE) {
+    return harden({});
```

Comment:
This is too loose. Once you see it is a remotable, you should also assert that it is an empty object before returning a new empty object. Otherwise, an accidental passing of a substantive remotable object will silently become an empty object rather than rejected with a nice diagnostic.

Reply:
@erights How would you assert it is an empty object if it's a presence? I thought doing something like getting the ownPropertyNames wouldn't work.

Reply:
See #1981